spooky-oysters/bangazon-web-app
# km-initial-models

## Description of Proposed Changes

- Brought in the boilerplate code from the bangazon boilerplate repo
- Added basic authentication (through the set-up file)

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout km-initial-models
```
In Visual Studio open the solution file
You should be able to build the solution without error
**I haven't run database migrations yet**


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Added models for PaymentType, Product, ProductType, Order, LineItem
* Data\ApplicationDbContext - updated to match boilerplate
* Data\DBInitializer - update to match boilerplate


## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
